tests/unittests/Foundation/NSStringTests.m

```diff
@@ -576,3 +576,41 @@ void testUrlCharacterSetEncoding(NSString* decodedString, NSString* encodedStrin
     string = @"/tmp/random.foo.tiff";
     ASSERT_TRUE([string.pathExtension isEqualToString:@"tiff"]);
 }
+
+TEST(NSString, Exceptions) {
+    NSRange range{ 100, 10 };
+
+    EXPECT_ANY_THROW([@"hello" characterAtIndex:10]);
```

Comment:
Thanks for checking.  A wise man once said "Its always good to ask".

---
In reply to: [71600230](https://github.com/Microsoft/WinObjC/pull/671#discussion_r71600230) [](ancestors = 71600230,71574787,71574294)